packages/node-http-handler/src/node-http-handler.ts

```diff
@@ -9,6 +9,7 @@ import { getTransformedHeaders } from "./get-transformed-headers";
 import { setConnectionTimeout } from "./set-connection-timeout";
 import { setSocketTimeout } from "./set-socket-timeout";
 import { writeRequestBody } from "./write-request-body";
+import { setSocketKeepAlive, SocketKeepAliveOptions } from "./set-socket-keep-alive";
```

Comment:
The SocketKeepAliveOptions import is no longer needed
```suggestion
import { setSocketKeepAlive } from "./set-socket-keep-alive";
```